src/signer/server.rs

```diff
@@ -63,42 +76,81 @@ impl Signer for Ed25519Signer {
 }
 
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn std::error::Error>> {
+async fn main() -> remote_signer::Result<()> {
```

Comment:
Result Error type

Reply:
No need it is an implicit `RemoteSignerError`